<?xml version="1.0"?>
<launch>

    <node pkg="cv_camera" type="cv_camera_node" name="center_camera">
        <param name="device_id" value="0"/>
        <param name="frame_id" value="center_camera"/>
        <param name="image_width" value="640"/>
        <param name="image_height" value="480"/>
        <!-- <param name="camera_info_url" value="file://$(env HOME)/.ros/camera_info/center_camera.yaml" />-->
    </node>

    <node pkg="cv_camera" type="cv_camera_node" name="left_camera">
        <param name="device_id" value="2"/>
        <param name="frame_id" value="left_camera"/>
        <param name="image_width" value="640"/>
        <param name="image_height" value="480"/>
        <!--<param name="camera_info_url" value="file://$(env HOME)/.ros/camera_info/left_camera.yaml" />-->
    </node>

    <node pkg="cv_camera" type="cv_camera_node" name="right_camera">
        <param name="device_id" value="1"/>
        <param name="frame_id" value="right_camera"/>
        <param name="image_width" value="640"/>
        <param name="image_height" value="480"/>
        <!--<param name="camera_info_url" value="file://$(env HOME)/.ros/camera_info/right_camera.yaml" />-->
    </node>

</launch>        